src/o365/todo/commands/list/list-list.ts

```diff
@@ -0,0 +1,58 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import GraphCommand from '../../../base/GraphCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions { }
+
+class TodoListListCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.LIST_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Returns a list of Microsoft To Do task lists';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+
+    const requestOptions: any = {
+      url: `${this.resource}/beta/me/todo/lists`,
+      headers: {
+        accept: 'application/json;odata.metadata=none',
+        'content-type': 'application/json'
+      },
+      json: true
+    };
+
+    request
+      .get(requestOptions)
+      .then((res: any): void => {
+        cmd.log(res.value);
```

Comment:
Let's not output all properties by default, because they won't fit the terminal window. Instead, for the default text output let's show just the displayName and id and then in output JSON we can show everything we got from the API.